packages/angular-meteor-data/tests/integration/reactive.spec.js

```diff
@@ -388,5 +460,35 @@ describe('angular-meteor.reactive', function() {
         expect(changedSpy).toHaveBeenCalled();
       });
     });
+
+    describe('$reactivateCollection()', function() {
+      var scope;
+      var vm;
+
+      beforeEach(function() {
+        scope = $rootScope.$new();
+        vm = scope.$viewModel({});
+      });
+
+      afterEach(function() {
+        scope.$destroy();
+      });
+
+      it('should return model', function() {
+        vm.myProp = 10;
+        expect(vm.$reactivateCollection('myProp')).toEqual(10);
+      });
+
+      it ('should register a scope collection watcher', function() {
+        vm.myProp = 'myProp';
+        var watchSpy = spyOn(scope, '$watchCollection');
+
+        vm.$reactivateCollection('myProp', false);
```

Comment:
`$reactivateCollection` does not have an `isDeep` argument, so this `false` parameter is extraneous.